{"uuid": "5ad57d14-8f17-4e99-98d2-b85e74125807", "children": ["5cca54a8-ca5b-4567-8f7f-649145f4a011"], "befores": [{"name": "login_fixture", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'boby'\n", "trace": "  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/testcase/conftest.py\", line 23, in login_fixture\n    return result.boby['token']\n"}, "start": 1740061223629, "stop": 1740061223815}], "start": 1740061223629, "stop": 1740061223899}